metadata_gui_panel.py

```
@@ -123,15 +168,20 @@ def __init__(self, comic_data: GUIComicInfo) -> None:
+        self.name = f"{metadata.title}: {metadata.title}"
 
         main_layout = QVBoxLayout()
         content_layout = QGridLayout()
 
         creators_box = DashboardBox("Creators")
         for role, people in metadata.creators:
-            creators_box.add_role_box(RoleBox(role, people))
+            box = RoleBox(role, people)
+            if box.is_empty:
+                continue
+            else:
```

Ruff (0.15.15)

[warning] Unnecessary `else` after `continue` statement

Remove unnecessary `else`

(RET507)